tests/StripeChargeControllerTest.php

```diff
@@ -5,12 +5,21 @@
 
 use Illuminate\Foundation\Testing\WithFaker;
 use Illuminate\Support\Facades\Config;
+use Stripe\Charge;
 use Stripe\StripeClient;
 
 class StripeChargeControllerTest extends TestCase
 {
     use WithFaker;
 
+    protected $stripe;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->stripe = new StripeClient(Config::get('services.stripe.secret'));
```

Comment:
What if we, in setUp, just said "if there's at least one charge, use that; otherwise create one and use that"